README.md

```diff
@@ -41,7 +41,7 @@ Then simply open [`http://localhost:3000`](http://localhost:3000) on your web br
 
 (with npm >= 2)
 ```
-$ npm start [-- STAGING_PATH [HTTP_BASE_URI [PORT]]]
+$ npm start [STAGING_PATH [HTTP_BASE_URI [PORT [RESULT_PATH]]]]
```

Comment:
As far as I understood, the `--` is mandatory, [as explained here](http://stackoverflow.com/a/14404223/1935861). Did you experience the opposite?